generate_ca_courts.py

```diff
@@ -0,0 +1,609 @@
+    },
+    {
+        "name": "Yuba",
+        "date": "1850-02-18",
+        "url": "https://www.yuba.courts.ca.gov/",
+        "id": "yub",
+    },
+]
+
+# Administrative tribunals
+ADMIN_TRIBUNALS = [
+    {
+        "id": "calstatebar",
+        "name": "State Bar Court of California",
+        "name_abbreviation": "Cal. State Bar Ct.",
+        "citation_string": "Cal. State Bar Ct.",
+        "court_url": "https://www.statebarcourt.ca.gov/",
+        "date": "1989-01-01",
+        "level": "special",
+        "type": "special",
+        "notes": "Disciplinary tribunal for California attorneys. Has Hearing Department (trial level) and Review Department (appellate level).",
+        "examples": [
+            "State Bar Court of California",
+            "California State Bar Court",
+            "State Bar Court Review Department",
+            "State Bar Court Hearing Department",
+        ],
+        "regex": [
+            "State Bar Court of California",
+            "California State Bar Court",
+            "State Bar Court Review Department",
+            "State Bar Court Hearing Department",
+            "State Bar Court",
+            "Cal\\.? State Bar Court",
+            "State Bar of California Court",
+        ],
+    },
+    {
+        "id": "calwcab",
+        "name": "Workers' Compensation Appeals Board of California",
+        "name_abbreviation": "Cal. WCAB",
+        "citation_string": "Cal. WCAB",
+        "court_url": "https://www.dir.ca.gov/wcab/wcab.htm",
+        "date": "1913-01-01",
+        "level": "special",
+        "type": "special",
+        "notes": "Adjudicates workers' compensation disputes in California.",
+        "examples": [
+            "Workers' Compensation Appeals Board",
+            "California WCAB",
+            "Cal WCAB",
+        ],
+        "regex": [
+            "Workers'? Compensation Appeals Board",
+            "California WCAB",
+            "Cal\\.? WCAB",
+            "WCAB",
+        ],
+    },
+    {
+        "id": "calcuiab",
+        "name": "California Unemployment Insurance Appeals Board",
+        "name_abbreviation": "Cal. CUIAB",
+        "citation_string": "Cal. CUIAB",
+        "court_url": "https://www.cuiab.ca.gov/",
+        "date": "1943-01-01",
+        "level": "special",
+        "type": "special",
+        "notes": "Hears appeals of unemployment and disability insurance decisions.",
+        "examples": [
+            "California Unemployment Insurance Appeals Board",
+            "CUIAB",
+        ],
+        "regex": [
+            "California Unemployment Insurance Appeals Board",
+            "Unemployment Insurance Appeals Board",
+            "CUIAB",
+        ],
+    },
+    {
+        "id": "calperb",
+        "name": "California Public Employment Relations Board",
+        "name_abbreviation": "Cal. PERB",
+        "citation_string": "Cal. PERB",
+        "court_url": "https://perb.ca.gov/",
+        "date": "1975-01-01",
+        "level": "special",
```

Comment:
levels should be ljc, gjc, iac, cola ... I think.   Unless there is something else.